app-nextjs-starter/pages/_document.tsx

```diff
@@ -1,5 +1,5 @@
-import * as React from 'react'
-import { Html, Head, Main, NextScript } from 'next/document'
+import * as React from "react";
+import { Html, Head, Main, NextScript } from "next/document";
```

Comment:
@demetriusfeijoo I think this repository is missing `.prettierrc` file, while it is inside `app-nuxtjs-3-starter/.prettierrc`. Could you create the one here, too?

Because we don't have it here, unnecessary changes are being made here. After copy&pasting the file here, you may need to re-format all these, to have smaller changes.

Ping me when you're done. Then I'll start reviewing this PR again.

Reply:
That's true @eunjae-lee. 

I've added the prettierrc file and also formatted all files as you suggested.